src/main/java/uk/gov/hmcts/sptribs/common/service/AuthorisationService.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.sptribs.common.service;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.idam.client.models.User;
+import uk.gov.hmcts.sptribs.idam.IdamService;
+
+import javax.servlet.http.HttpServletRequest;
+
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+import static uk.gov.hmcts.sptribs.common.config.ControllerConstants.BEARER_PREFIX;
+
+@Service
+@Slf4j
+public class AuthorisationService {
```

Comment:
As the supplementary data changes are not required, Authorisation service can be removed as well.

Reply:
Removed the file from PR.